src/testsuite/arith128_test_i64.c

```diff
@@ -393,102 +452,233 @@ test_vsrad (void)
 #undef __DEBUG_PRINT__
 
 int
-test_permdi (void)
+test_mrghld (void)
```

Comment:
dang, the functions no longer line up, making review a little more challenging.  consider adding new functions to the end of existing files?

Reply:
well ... the file order is often driven by copy paste of related similar test cases /constants. I will try to better in the future. But dont see a need to change this now.